### PR TITLE
Add an explanation of subcommands

### PR DESCRIPTION
**Pull request by baruchiro**
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).

I searched for a record for a `git checkout` and nearly opened an issue on it.
I think if there is a user who is less engaged in open source, he would not know how to get records for subcommands.

### CODE REVIEW

File: pages/common/tldr.md [hunk 1]
```diff
@@ -5,3 +5,7 @@
 - Get typical usages of a command (hint: this is how you got here!):
 
 `tldr {{command}}`
+
+- For typical usages of a subcommand:
+
+`tldr {{command-subcommand}}`
```
Comment (agnivade) [inline pages/common/tldr.md]: This is not true. You don't need the `-`. You can do "tldr git merge" and it works fine.

File: pages/common/git.md [hunk 2]
```diff
@@ -14,6 +14,10 @@
 
 `git help {{command}}`
 
+- Call tldr on a command:
+
+`tldr {{git-command}}`
```
Comment (agnivade) [inline pages/common/git.md]: There should not be a tldr command in a git page.

File: pages/common/tldr.md (hunk 1 above)
Comment (baruchiro) [reply pages/common/tldr.md]:
```bash
$ tldr git merge
Invalid argument `merge'

Usage: tldr [-v|--version] [--update] COMMAND
  tldr Client program

```

Ubuntu 18.10

File: pages/common/git.md (hunk 2 above)
Comment (baruchiro) [reply pages/common/git.md]: I understand, but when a user search for help with `git checkout`, how he can find the result?
If you say that he gets the result from `tldr tldr`, I will accept.

Comment (Accurate0) [reply pages/common/git.md]: $ tldr git-checkout

git checkout

Checkout a branch or paths to the working tree.
...

Although most clients automatically add a "-" between words.

Comment (baruchiro) [reply pages/common/git.md]: @Accurate0 , I know this now, but

> I searched for a record for a `git checkout` and nearly opened an issue on it.
> I think if there is a user who is less engaged in open source, he would not know how to get records for subcommands.

Just after traveling in this repo, I found the solution. So if there is a user that not familiar with Github or open-source, how he would find the solution?

File: pages/common/tldr.md (hunk 1 above)
Comment (agnivade) [reply pages/common/tldr.md]: That is a client specific thing. The node client handles it fine. Since this is not in any spec, I don't think we should mention it.

File: pages/common/git.md (hunk 2 above)
Comment (agnivade) [reply pages/common/git.md]: The most basic tendency is to write the exact command; which in this case is "git checkout". So if you do "tldr git checkout", it should give the correct result. And the node client indeed does that. So I don't see any confusion here.

Now, if some client does not implement this feature, and the require you to pass a `-` or whatever, I would expect the person to file an issue with the appropriate client repo asking what the solution is.

Please note that this repo is just for pages, and we do not control the client authors. There are lots of clients in lots of programming languages. If you have any confusion regarding how to find a command, you should ask that client's owner.

Does that clarify things ?

File: pages/common/tldr.md (hunk 1 above)
Comment (sbrl) [reply pages/common/tldr.md]: So does the bash client. Perhaps you could open an issue against your client's repo?

_(For my own reference, this is worth including in the official client spec that's being drafted in #1065)_

File: pages/common/git.md (hunk 2 above)
Comment (baruchiro) [reply pages/common/git.md]: Yes, Thank you
